### ISSUE TITLE
Improve Enum Textarea Placeholder

### ISSUE THREAD

**Issue by carsonbird** [bjdash/JSON-Schema-Builder#5]
The Enum Placeholder uses single quotes (`'`) as an example, but the system only includes the enum property in the raw JSON export if you use double quotes (`"`). I'd recommend fixing this inconsistency.

**Comment by linonetwo** [comment]
Yes, if use `'`, this feature is not working. Change to `"` works, thanks!
